app/core/util.py

```diff
@@ -73,3 +77,147 @@ def tree_table_to_json(
             append_list.append(node_row_object)
 
     return json_out
+
+
+# TODO move when we have a better idea of how we want to handle this
+class DfcRow:
```

Comment:
Assuming we will bring @diversemix row object as well as some of the other funtionality from the scripts into the backend.